src/Runner.Common/Logging.cs

```diff
@@ -40,7 +39,6 @@ public override void Initialize(IHostContext hostContext)
         {
             base.Initialize(hostContext);
             _totalLines = 0;
-            _pageId = Guid.NewGuid().ToString();
```

Comment:
we don't really need a new guid for each page, we can leverage the timeline record id for each steps, those are already unique per step.